docs/operation.md

```diff
@@ -0,0 +1,1100 @@
+ "type": “Appointment",
+ "id": "1620549"
+}
+```
+
+**Related cerner documentation**
+[https://fhir.cerner.com/millennium/dstu2/scheduling/appointment/#update](https://fhir.cerner.com/millennium/dstu2/scheduling/appointment/#update)
+
+## Search Allergy Intolerance of a patient
+The searchAllergyIntolerance operation search Allergy intolerance reports of a particular patient.
+
+**searchAllergyIntolerance**
+```xml
+<cerner.searchAllergyIntolerance>
+    <base>{$ctx:base}</base>
+    <type>{$ctx:type}</type>
+    <id>{$ctx:id}</id>
+    <patient>{$ctx:patient}</patient>
+    <status>{$ctx:status}</status>
+</cerner.searchAllergyIntolerance>
+
+```
+**Properties**
+* base: The service root URL.
+* type: The type of the resource.
+* id: The logical Id of the Allergy Intolerance.
+* patient: patient who the sensitivity is for.
+* status[optional]: The certainty of the allergy or intolerance,Example: confirmed.
+
+>Note: One of the id or patient must be provided.
+
+**Sample request**
+
+Following is a sample REST request that can be handled by the searchAllergyIntolerance operation.
+```
+{
+ "base": “https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/",
+ "type": “AllergyIntolerance",
+ "id": "5703737",
+ "status":"active"
+}
+```
+
+**Related cerner documentation**
+
+[https://fhir.cerner.com/millennium/dstu2/general-clinical/allergy-intolerance/#search](https://fhir.cerner.com/millennium/dstu2/general-clinical/allergy-intolerance/#search)
+ 
+## Search Appointments scheduled
+The Appointment operation provides information about scheduled appointments.
+
+**searchAppointment**
+```xml
+<cerner.searchAppointment>
+    <base>{$ctx:base}</base>
+    <type>{$ctx:type}</type>
+    <id>{$ctx:id}</id>
+    <date>{$ctx:date}</date>
+    <startDate>{$ctx:startDate}</startDate>
+    <endDate>{$ctx:endDate}</endDate>
+    <patient>{$ctx:patient}</patient>
+    <practitioner>{$ctx:practitioner}</practitioner>
+    <location>{$ctx:location}</location>
+    <status>{$ctx:status}</status>
+    <count>{$ctx:count}</count>
+</cerner.searchAppointment>
+```
+**Properties**
+* base: The service root URL.
+* type: The type of the resource.
+* id: The logical resource id associated with the resource
+* date: The Appointment date/time, if user specify the startDate and endDate, this parameter should not be used.
+* startDate: starting date of the Appointment(when searched using date range).
+* endDate: end date of the Appointment(when searched using date range).
+* patient: A single or comma separated list of Patient references.
+* practitioner: A single or comma separated list of Practitioner references.
+* location: A single or comma separated list of Location references.
+* status[optional]: A single or comma separated list of appointment statuses
+* count[optional]: The maximum number of results to return.
+
+>Note: id or one of patient, practitioner, or location must be provided.
```

Comment:
Shall we change this note as follows?
id or one of patient **or** practitioner, or location must be provided.